### PR TITLE
Fix width styling of Advisor daily email metrics cards

### PR DESCRIPTION
**Pull request by mhuth**
Add some extra tests of the HTML output as well.

### CODE REVIEW

**Comment by mhuth** [comment]
Before ...
before

After ...
Screenshot from 2022-12-20 12-52-20

**Comment by mhuth** [comment]
I had to get a funky with the Qute notation.  

I wanted to do this ...
`{#let width100=((new_recs && resolved_recs) || (new_recs && deact_recs) || (resolved_recs && deact_recs))}`

... but that didn't seem to work correctly so I had to do this instead to get the results I was after ... 

`{#let width100=(new_recs && resolved_recs)}{#let width100=(new_recs && deact_recs || width100)}{#let width100=(resolved_recs && deact_recs || width100)}`

/shrug

**Comment by mhuth** [comment]
@gwenneg @josejulio any chance you guys can look at this and merge it if it seems ok?  Thanks

**Comment by gwenneg** [comment]
I'll take a look at it today.

File: engine/src/main/resources/templates/Advisor/dailyEmailBody.html [hunk 1]
```diff
@@ -8,8 +8,9 @@
 <!-- Cards/metrics section -->
 <tr>
   <td>
-    <table role="presentation" border="0" cellpadding="0" cellspacing="0"
-     style="width: {#if action.context.advisor.size == 1}50{#else}100{/if}%; table-layout: fixed;">
+    {#let width100=(new_recs && resolved_recs)}{#let width100=(new_recs && deact_recs || width100)}{#let width100=(resolved_recs && deact_recs || width100)}
+    <table role="presentation" border="0" cellpadding="0" cellspacing="0" style="width: {#if width100}100{#else}50{/if}%; table-layout: fixed;">
+    {/let}{/let}{/let}
```
Comment (gwenneg) [inline engine/src/main/resources/templates/Advisor/dailyEmailBody.html:13]: Does this produce the output you need?
```suggestion
    <table role="presentation" border="0" cellpadding="0" cellspacing="0" style="width: {#if (new_recs && resolved_recs) || (new_recs && deact_recs) || (resolved_recs && deact_recs)}100{#else}50{/if}%; table-layout: fixed;">
```

Comment (mhuth) [reply engine/src/main/resources/templates/Advisor/dailyEmailBody.html]: Yes, that does the trick!  Thanks @gwenneg !  Now why didn't I think of that :/

**Review by gwenneg — APPROVED** [review]
LGTM

**Pull request merged**